packages/core/src/utils.ts

```diff
@@ -1544,7 +1544,7 @@ export const isCreateActionInput = (
     typeof create.value === 'string' &&
     typeof create.txData === 'string' &&
     typeof create.gas === 'string' &&
-    create.operation === Operation.Call &&
+    create.operation === Operation.DelegateCall &&
```

Comment:
This is irrelevant to the PR, but I noticed that the `Operation` was incorrect. This doesn't impact anything because we don't currently use this function